pkgs/servers/sql/cockroachdb/cockroachdb-bin.nix

```diff
@@ -28,7 +28,7 @@ let
 
 in
 buildFHSEnv {
-  inherit name;
```

Comment:
https://github.com/NixOS/nixpkgs/pull/257316 was merged so now it's possible to just `inherit pname version`

Reply:
Neat, rebased and changed it as such.